### PR TITLE
updates sponsorships page, addresses #250

### PR DESCRIPTION
**Pull request by arshiamufti**
Addresses #250.

### CODE REVIEW

**Comment by annalorimer** [comment]
lgtm

File: content/pages/sponsors.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+Occasionally, companies will help us run events by providing us with funding. In the past, we have
+worked on the following projects.
 
 ## Technical Workshop Series 
 #### Brought to you in partnership with ![BlackBerry](/images/bb_logo.jpg)
 
-Currently we are running a Technical Workshop Series, which will consist of three
-technical skills workshops over the next three terms.
+During the 2015-2016 academic year, we ran a Technical Workshop Series, which consisted of three
+technical skills workshops over three terms. The events we ran are:
```
Comment (evykassirer) [inline content/pages/sponsors.md]: Are -> were

**Comment by evykassirer** [comment]
I think some of your lines are too long

**Comment by arshiamufti** [comment]
fixed the tense @evykassirer but I think the lines are correct--is it not rendering correctly?

File: content/pages/sponsors.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
 Title: Sponsorship
 status: hidden
 
-Occasionally companies will help us run events by providing us with funding, currently we are working
-on the following projects:
-
+Occasionally, companies will help us run events by providing us with funding. In the past, we have
```
Comment (evykassirer) [inline content/pages/sponsors.md]: this looks like >80chars

File: content/pages/sponsors.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+worked on the following projects.
 
 ## Technical Workshop Series 
 #### Brought to you in partnership with ![BlackBerry](/images/bb_logo.jpg)
 
-Currently we are running a Technical Workshop Series, which will consist of three
-technical skills workshops over the next three terms.
+During the 2015-2016 academic year, we ran a Technical Workshop Series, which consisted of three
```
Comment (evykassirer) [inline content/pages/sponsors.md]: same here

**Comment by annalorimer** [comment]
@arshiamufti The lines are too long by about 18 characters in some places

**Pull request merged**